### DISCUSSION TITLE
Homing issue

### DISCUSSION THREAD

**Discussion by beadmaker** [arkypita/LaserGRBL#1419] [Q&A]
I’m using 1.1h, LaserGRBL 4.3. It says MSG: Check Limits, G code lock when I try homing. What do I need to do as I am new to GRBL?

**Comment by gmmanonymus111** [comment]
That message means, at least one of your limit switches is triggered. Make sure none of them is pressed, you wired them correctly, and there is no too much electrical noise while homing.
